tests/test.toml

```diff
@@ -38,16 +38,6 @@ player_score=8
 crib_score=8
 name="two pair"
 
-[[tests]]
```

Comment:
This was a dupe of the 2nd test. Ended up causing problems in the name generation for the test case, since the names are the same.